packages/components/src/utils/select/select-dropdown.ts

```diff
@@ -3,7 +3,7 @@ import { OPTION_HEIGHT } from '../../styles/option-styles';
 export const SELECT_DROPDOWN_DIRECTIONS_INTERNAL = ['down', 'up'] as const;
 export type SelectDropdownDirectionInternal = (typeof SELECT_DROPDOWN_DIRECTIONS_INTERNAL)[number];
 export const SELECT_DROPDOWN_DIRECTIONS = [...SELECT_DROPDOWN_DIRECTIONS_INTERNAL, 'auto'] as const;
-export type SelectDropdownDirection = (typeof SELECT_DROPDOWN_DIRECTIONS)[number];
+export type SelectDropdownDirections = (typeof SELECT_DROPDOWN_DIRECTIONS)[number];
```

Comment:
Should stay singular since when the type is used it is a single value, not an array.